exporter/collector/integrationtest/testcases/testcases_metrics.go

```diff
@@ -229,5 +231,78 @@ var MetricsTestCases = []TestCase{
 			cfg.MetricConfig.ClientConfig.Compression = "gzip"
 		},
 	},
+	{
+		Name:                 "Write ahead log enabled",
+		OTLPInputFixturePath: "testdata/fixtures/metrics/basic_counter_metrics.json",
+		ExpectFixturePath:    "testdata/fixtures/metrics/basic_counter_metrics_wal_expect.json",
+		CompareFixturePath:   "testdata/fixtures/metrics/basic_counter_metrics_expect.json",
```

Comment:
"expect" is what the input for a test converts into, and it verifies that a single test hasn't changed since its last fixture was checked in

"compare" is to cross-reference output fixtures between different tests that should have equal outcomes. In this case, we want to make sure that enabling WAL gives the same output as normal export, based on the same input fixture.

Reply:
there is also a comment in the definition in testcase.go:

```
 	// CompareFixturePath is a second output fixture that should be equal to this test's output fixture.
	// Used for cross-referencing multiple tests that should have the same output, without overwriting the same fixtures.
```